index.js

```diff
@@ -55,8 +56,9 @@ module.exports = {
         var manifestPath  = this.readConfig('manifestPath');
         var cacheControl  = this.readConfig('cacheControl');
         var expires       = this.readConfig('expires');
+        var dotFolders     = this.readConfig('dotFolders');
```

Comment:
Very very minor nitpick, but mind lining that `=` sign up for us please @habdelra ? ;)